src/utils.rs

```diff
@@ -404,28 +411,39 @@ pub(crate) async fn launch(
     println!("{} was selected", exe.display());
 
     #[cfg(not(target_os = "windows"))]
-    let should_use_wine = os == &BuildOs::Windows;
-    let (binary, args) = (
-        #[cfg(target_os = "windows")]
-        exe.to_str().unwrap().to_owned(),
-        #[cfg(not(target_os = "windows"))]
-        if should_use_wine {
-            wine_bin.unwrap().to_str().unwrap().to_owned()
-        } else {
-            exe.to_str().unwrap().to_owned()
-        },
-        #[cfg(target_os = "windows")]
-        "".to_owned(),
-        #[cfg(not(target_os = "windows"))]
-        if should_use_wine {
-            exe.to_str().unwrap().to_owned()
+    let should_use_wine = (os == &BuildOs::Windows) && !no_wine;
+    let wrapper_string = if wrapper.is_some() {
+            wrapper.unwrap_or_default().to_str().unwrap().to_owned()
         } else {
             "".to_owned()
-        },
-    );
+        };
+
+    let wrapper_vec = if !wrapper_string.is_empty() {
+        split(&wrapper_string.to_owned()).unwrap()
+    } else {
+        Vec::<String>::new()
+    };
+    let binary = 
+        if wrapper_vec.len() > 0 {
+            wrapper_vec[0].to_owned()
+        } else {
+            if should_use_wine {
+                wine_bin.unwrap().to_str().unwrap().to_owned()
+            } else {
+                exe.to_str().unwrap().to_owned()
+            }
+        };
 
     let mut command = tokio::process::Command::new(binary);
-    command.arg(args);
+    if wrapper_vec.len() > 1 {
+        for val in wrapper_vec.iter().skip(1) {
+            command.arg(val);
+        };
+    };
+
+    if !wrapper_string.is_empty() || should_use_wine == true {
```

Comment:
Why the `== true` here?

Reply:
Oh, was trying out something and forgot to delete, removed